src/Command/ContentHubServiceUuid.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Acquia\Console\ContentHub\Command;
+
+use Acquia\Console\ContentHub\Client\ContentHubCommandBase;
+use Acquia\Console\Helpers\Command\PlatformCmdOutputFormatterTrait;
+use EclipseGc\CommonConsole\Command\PlatformBootStrapCommandInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class ContentHubServiceUuid.
+ */
+class ContentHubServiceUuid extends ContentHubCommandBase implements PlatformBootStrapCommandInterface {
+
+  use PlatformCmdOutputFormatterTrait;
+
+  /**
+   * {@inheritDoc}
+   */
+  protected static $defaultName = 'ach:get-ch-service-uuid';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function configure() {
+    $this->setDescription('Checks for Content Hub module 2.x version.');
+    $this->setAliases(['ach-chuuid']);
+    $this->setHidden('TRUE');
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  public function execute(InputInterface $input, OutputInterface $output) {
+    $module_version = $this->drupalServiceFactory->getModuleVersion();
+    if ($module_version === 2) {
+      $remote_settings = $this->achClientService->getRemoteSettings();
+      $uuid = $remote_settings['uuid'];
```

Comment:
Is it possible that the $remote_settings array will not have a 'uuid' key??? in this case it would fail with undefined index warning.

Reply:
Added a check.